public/docs/ts/latest/quickstart.jade

```diff
@@ -154,89 +154,90 @@ block install-packages
-
-    **Create #{_an} #{_appDir} subfolder** off the project root directory:
-
-  code-example.code-shell.
-    mkdir #{_appDir}
-
-  :marked
-    Create the file `app/app.module.ts` with the following content:
-
-  +makeExample('app/app.module.1.ts')(format='.')
-
-  :marked
-    This is the entry point to your application.
-
-    Since the QuickStart application is a web application that runs in a browser, your root module
-    needs to import the
-    [`BrowserModule`](../latest/api/platform-browser/index/BrowserModule-class.html)
-    from `@angular/platform-browser` to the `imports` array.
-
-    This is the smallest amount of Angular that is needed for a minimal application to run in the
-    browser.
-
-    The QuickStart application doesn't do anything else, so you don't need any other modules. In a real
-    application, you'd likely import [`FormsModule`](../latest/api/forms/index/FormsModule-class.html) 
-    as well as [`RouterModule`](../latest/api/router/index/RouterModule-class.html) and
-    [`HttpModule`](../latest/api/http/index/HttpModule-class.html). These are introduced in the
-    [Tour of Heroes Tutorial](./tutorial/).
 
 .l-main-section
 h1#root-component Step !{step++}: Create a component and add it to your application
 :marked
   Every Angular application has at least one component: the _root component_, named `AppComponent`
-  here.
+  by convention.
 
   Components are the basic building blocks of Angular applications. A component controls a portion
-  of the screen&mdash;a *view*&mdash;through its associated template.
+  of the screen &mdash; a *view* &mdash; through its associated template. 
+  This QuickStart has just one component.
 
 a#app-component
+
+p.
+  #[b Create #{_an} #{_appDir} subfolder] off the project root directory
+
+code-example.code-shell.
+  mkdir #{_appDir}  
 p.
   #[b Create the component file]
   #[code #[+adjExPath('app/app.component.ts')]] with the following content:
 
 +makeExample('app/app.component.ts')
 
 :marked
-  The QuickStart application has the same essential structure as any other Angular component:
+  The QuickStart `AppComponent` has the same essential structure as any other Angular component:
 
     * **An import statement**. Importing gives your component access to
     Angular's core [`@Component` decorator function](../latest/api/core/index/Component-decorator.html).
+
     * **A @Component #{_decorator}** that associates *metadata* with the
         `AppComponent` component class:
 
         - a *selector* that specifies a simple CSS selector for an HTML element that represents
           the component.
         - a *template* that tells Angular how to render the component's view.
+      +ifDocsFor('ts')
+        - *backticks* (`) surrounding the template HTML.
+        Use backticks instead of quote marks (' or ") 
+        so the HTML can span multiple lines as the template grows.
+
     * **A component class** that controls the appearance and behavior of a view
```

Comment:
Frankly this one sentence doesn't "teach concepts" more than _any of the many other sentences around it_.  I don't understand why the other statements are valuable and this one is not.

We can delete the comment. If we do I suggest we remove the backticks too  ... even though experience teaches us that it is easier to add HTML to the template when you start with backticks.